Classes/Service/DeeplService.php

```diff
@@ -69,14 +67,16 @@ public function translateContent(TranslateContext $translateContext): string
         }
         // If the source language is set to Autodetect, no glossary can be detected.
         if ($translateContext->getSourceLanguageCode() !== null) {
-            // @todo Make glossary findable by current site.
-            $glossary = $this->glossaryRepository->getGlossaryBySourceAndTarget(
+            /** @var GlossaryKeyEvent $glossaryEvent */
```

Comment:
This docblock should be not needed. Otherwise we may need to add some phpstorm meta data stuff.